integration-tests/scripts/.env.template

```
@@ -15,12 +15,13 @@ export TSSC_APP_DEPLOYMENT_NAMESPACES=""
 
 #### Github
 # https://docs.redhat.com/en/documentation/red_hat_trusted_application_pipeline/1.2/html-single/installing_red_hat_trusted_application_pipeline/index#creating_a_github_personal_access_token
-export GITHUB__APP__ID=
-export GITHUB__APP__CLIENT__ID=
-export GITHUB__APP__CLIENT__SECRET=
-export GITHUB__APP__PRIVATE_KEY=
-export GITOPS__GIT_TOKEN=
-export GITHUB__APP__WEBHOOK__SECRET=
+export GITHUB_APP_ID=
+export GITHUB_APP_CLIENT__ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_APP_CLIENT__ID key should go before the GITHUB_APP_ID key

(UnorderedKey)